src/utils/positions.ts

```
@@ -10,6 +12,7 @@ import {
   WarningWithDetail,
   UserRebalancerInfo,
 } from './types';
+import { getClient } from './rpc';
```

ESLint

[error] `./rpc` import should occur before import of `./types`

(import/order)